gemp-lotr/gemp-lotr-async/src/main/web/src/updated/jCard/modules/sites/types/range-generation.functions.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { generateRangeOfNumbers } from "./range-generation.functions";
+
+describe('range-generation.functions', () => {
```

Comment:
if you build something to depend on it, even if it's silly, write some tests.

specifically worried about off by one issues due tp array indexing